### ISSUE TITLE
filter HTTP_PROXY

### ISSUE THREAD

**Issue by bef** [stefanesser/suhosin#108]

**Comment by NewEraCracker** [comment]
Just for the record this is much easier to be mitigated at Webserver level: https://httpoxy.org/

**Comment by bef** [comment]
Correct. In fact most of Suhosin's filtering capabilities can be easily replaced by other filters either using the web server (mod_rewrite, mod_env, ...) or by configuring a WAF (e.g. mod_security). It just takes a lot of time and effort to configure all of them, so why not add some important filters to Suhosin.
